packages/admin/cms-admin/package.json

```diff
@@ -151,6 +151,7 @@
         "@emotion/styled": "^11.3.0",
         "@mui/material": "^5.0.0",
         "@mui/x-data-grid": "^5.0.0",
+        "@mui/x-data-grid-pro": "^5.0.0",
```

Comment:
In the admin package we made this peer dependency optional, would this work here as well? https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/package.json#L119-L124

Reply:
Cool! I think so, added in 6ca0fc6cda80e37ca33c95d253fe2e505e52e539